src/ape/managers/converters.py

```diff
@@ -91,7 +92,67 @@ def convert(self, value: str) -> AddressType:
         return to_checksum_address(value)
 
 
-hex_address_converter = HexAddressConverter(None, None)  # type: ignore
+hex_address_converter = HexAddressConverter(None, None, None)  # type: ignore
+
+
+class ListTupleConverter(ConverterAPI):
+    """
+    A converter that converts any tuple or list of items
+    :class:`~ape.types.AddressType`.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        return isinstance(value, list) or isinstance(value, tuple)
+
+    def convert(self, value: Union[list, tuple]) -> Union[list, tuple]:
```

Comment:
```suggestion
    def convert(self, value: Union[lList, Tuple]) -> Union[List, Tuple]:
```

Don't these have to be the `typing` classes?